helm/applications/skaha/Chart.yaml

```diff
@@ -21,7 +21,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.3.2
+version: 1.3.3
```

Comment:
Are these skaha changes intended to be included too?

Reply:
Yes, they're independent of this.  I added resources to the `initContainer` to make sure they weren't influencing anything.

Reply:
At the end of the day the initContainer runs in the same cgroup with the same effective resource requests and limits (even if not declared) as the overall pod, which are determined based on this logic:  https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#resource-sharing-within-containers

So it's not possible for initContainers to be "smaller" than the other containers in the pod.